src/tz.cpp

```diff
@@ -29,7 +29,10 @@
 // Our apologies.  When the previous paragraph was written, lowercase had not yet
 // been invented (that would involve another several millennia of evolution).
 // We did not mean to shout.
-
+#ifdef RTC
+#  include "date/embedded_data_file.h"
+#  include "date/ptz.h"
+#endif
```

Comment:
If def'd these needed includes to make it obvious this is RTC configuration. These are the first non-copywrite lines in the file so the extra length seemed worth it.